flink-runtime-web/web-dashboard/src/app/pages/job/job.module.ts

```diff
@@ -53,7 +56,10 @@ import { NzPopconfirmModule } from 'ng-zorro-antd/popconfirm';
     NzDividerModule,
     NzCollapseModule,
     NzEmptyModule,
+    NzIconModule,
```

Comment:
FYI: I realized that the icon in the exception history was also not rendered. That's why I noticed that we should also add the Icon module